spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe User do
+  describe 'Associations' do
+    it { should have_many(:manifests) }
+  end
+
+  describe 'cdx_user_id' do
+    it 'is always required' do
+      user = create(:user)
+      expect(user.cdx_user_id).to_not be_nil    
+    end
+
+    it 'is unique' do
+      user = create(:user)
+      expect {
+        create(:user, cdx_user_id: user.cdx_user_id)
```

Comment:
I think for this you could just use shoulda for this: `it { should validate_uniqueness_of(:cdx_user_id) }`